### PR TITLE
[Her-118] Change cameraId filter to list

### PR DESCRIPTION
**Pull request by TBiannca**
## Related
[HER-118](https://nerds-interns.atlassian.net/browse/HER-118?atlOrigin=eyJpIjoiMzIzMjk2YmExYmI1NGU3M2I0MjEzNWFkNTJlODAwMmYiLCJwIjoiaiJ9)

### CODE REVIEW

File: src/Data/Detection/Filtering/DetectionFilter.cs [hunk 1]
```diff
@@ -23,7 +22,7 @@ public IQueryable<EDetection> Execute(IQueryable<EDetection> input)
             input = MatchClass(input);
             input = MatchTimestampLesserThan(input);
             input = MatchTimestampGreaterThan(input);
```
Comment (miutamihai) [inline src/Data/Detection/Filtering/DetectionFilter.cs:24]: Let's leave an empty line before our return statements :smile:

File: src/Seeds/Data/cameras.json [hunk 2]
```diff
@@ -1,6 +1,17 @@
-[{
-  "id": "default",
-  "latitude": "53.27137",
-  "longitude": "-7.827415"
-}
+[
+  {
+    "id": "default",
```
Comment (miutamihai) [inline src/Seeds/Data/cameras.json:3]: Is adding more cameras to the seed really necessary? If yes, maybe we should use different values for our entries.

**Review by miutamihai — CHANGES_REQUESTED** [review]

File: src/Data/Detection/Filtering/DetectionFilter.cs (hunk 1 above)
Comment (TBiannca) [reply src/Data/Detection/Filtering/DetectionFilter.cs:24]: done

File: src/Seeds/Data/cameras.json (hunk 2 above)
Comment (TBiannca) [reply src/Seeds/Data/cameras.json:3]: done

Comment (miutamihai) [reply src/Seeds/Data/cameras.json:3]: After talking with @danidoff  we decided that we should keep just one camera inside of our seed, so the comment I left becomes invalid. Let's make this change and then we're good.

Comment (TBiannca) [reply src/Seeds/Data/cameras.json:3]: done

**Review by miutamihai — APPROVED** [review]
:rocket:

**Review by danidoff — APPROVED** [review]

**Pull request merged**